README.md

```diff
@@ -23,6 +23,87 @@ gem install optimizely-sdk
 To access the Feature Management configuration in the Optimizely dashboard, please contact your Optimizely account executive.
 
 ### Using the SDK
+
+To use the SDK, the Optimizely instance can be initialized in three different ways as per your requirement. Ruby SDK also includes a factory class, OptimizelyFactory, which provides methods to create Optimizely instance with default configuration.
```

Comment:
Suggest revising to:

You can initialize the Optimizely instance in three ways: with a datafile, by providing an `sdk_key`, or by providing a Config Manager. Each method is described below. The Ruby SDK also includes a factory class, `OptimizelyFactory`, which provides methods to create an Optimizely instance with the default configuration.